djangocms_versioning/admin.py

```diff
@@ -99,6 +104,19 @@ def content_link(self, obj):
     content_link.short_description = _('Content')
     content_link.admin_order_field = 'content'
 
+    def state_actions(self, obj):
+        """Display links to state change endpoints
+        """
+        archive_url = reverse('admin:djangocms_versioning_{model}_archive'.format(
+            model=self.model.__name__.lower(),
```

Comment:
Use `obj._meta.app_label` and `obj._meta.model_name`

Reply:
`obj._meta.model_name` doesn't actually work because it seems to return the non-proxy model name